pkg/apiserver/jsonpatch/jsonpatch.go

```diff
@@ -0,0 +1,50 @@
+package jsonpatch
+
+import (
+	"encoding/json"
+)
+
+type PatchOperation struct {
+	Op    string      `json:"op,omitempty"`
+	Path  string      `json:"path,omitempty"`
+	Value interface{} `json:"value,omitempty"`
+}
```

Comment:
I was going to complain about this, but it appears that jsonpatch doesn't actually provide something nice here.  Wild.